src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/manuell/brevmottaker/ManuellBrevmottakerService.kt

```diff
@@ -75,6 +86,52 @@ class ManuellBrevmottakerService(
             opprettetTidspunkt = LocalDateTime.now()
         )
     }
+
+    @Transactional
+    fun opprettBrevmottakerSteg(behandlingId: UUID) {
+        val behandling = behandlingRepository.findByIdOrThrow(behandlingId)
+        validerStegopprettelse(behandling)
+        behandlingskontrollService.behandleBrevmottakerSteg(behandlingId)
+    }
+
+    @Transactional
+    fun fjernManuelleBrevmottakereOgTilbakeførSteg(behandlingId: UUID) {
+        hentBrevmottakere(behandlingId).forEach { manuellBrevmottaker ->
+            manuellBrevmottakerRepository.deleteById(manuellBrevmottaker.id)
+            historikkService.lagHistorikkinnslag(
+                behandlingId = behandlingId,
+                historikkinnslagstype = TilbakekrevingHistorikkinnslagstype.BREVMOTTAKER_FJERNET,
+                aktør = Aktør.SAKSBEHANDLER,
+                opprettetTidspunkt = LocalDateTime.now()
+            )
+        }
```

Comment:
Hadde vært digg hvis man kunne ha gjenbrukt `fjernBrevmottaker` som ligger i denne klassen for å erstatte denne blokka, men ser at den henter opp brevmottaker igjen så det er vel mindre effektivt 😭

Reply:
Trakk ut blokka i en felles funksjon 👍